ecommerce/views_test.py

```diff
@@ -693,6 +720,29 @@ def test_patch_basket_update_active_inactive_product(basket_client, basket_and_c
     assert resp.status_code == status.HTTP_200_OK
 
 
+def test_patch_basket_update_active_inactive_product_readable_id(
```

Comment:
```suggestion
def test_patch_basket_update_inactive_product(
```

Reply:
We can't use that name because a test case is already there with the same name.